templates/common/paths.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+// ABCInternalDir is the name for internal directories that have things like
+// manifests and .gitkeep files. These may be destination directories or
+// golden-test data/ directories.
```

Comment:
nits is this a typo for `golden-test data/`

Reply:
I think it's good as written